src/benchmark_register.cc

```diff
@@ -203,6 +203,7 @@ Benchmark::Benchmark(const char* name)
     : name_(name),
       aggregation_report_mode_(ARM_Unspecified),
       time_unit_(kNanosecond),
```

Comment:
Can't you just `time_unit_(GetDefaultTimeUnit())`?

Reply:
Yes, that is perfectly reasonable.
1667d8e353d7c0abb3298883c53d9ec6470bf9cf